cgmanifest.json

```diff
@@ -24013,29 +24013,29 @@
       "component": {
         "type": "other",
         "other": {
-          "name": "python-requests_ntlm",
-          "version": "1.1.0",
-          "downloadUrl": "https://github.com/requests/requests-ntlm/archive/v1.1.0/requests_ntlm-1.1.0.tar.gz"
+          "name": "python-requests-oauthlib",
+          "version": "1.2.0",
+          "downloadUrl": "https://github.com/requests/requests-oauthlib/archive/v1.2.0.tar.gz"
         }
       }
     },
     {
       "component": {
         "type": "other",
         "other": {
-          "name": "python-requests-oauthlib",
-          "version": "1.2.0",
-          "downloadUrl": "https://github.com/requests/requests-oauthlib/archive/v1.2.0.tar.gz"
+          "name": "python-requests-toolbelt",
```

Comment:
These changes are just a reordering of a few entries to keep them in alphabetical order.